Homework-06-documentation-tests.ipynb

```diff
@@ -57,6 +57,25 @@
     "print(stat.count_words())\n"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import unittest\n",
+    "\n",
+    "class TestStringMethods(unittest.TestCase):\n",
+    "    def test_init(self):\n",
+    "        s = \"one one one two two three\"\n",
+    "        test = TextStats(s)\n",
+    "        \n",
+    "    def count_words(self):\n",
+    "        self.assertEqual(test.count_words,{'one':3,'two':2,'three':1})\n",
+    "\n",
+    "unittest.main()"
```

Comment:
you would also write test cases for other situations such as the string is empty and there are commas and periods in the text to make sure your method behaves the way you want it to in all possible scenarios.